pkg/cli/cmds/etcd_snapshot.go

```diff
@@ -24,19 +27,14 @@ func NewEtcdSnapshotCommand(action func(*cli.Context) error) cli.Command {
 				Destination: &ServerConfig.DataDir,
 			},
 			&cli.StringFlag{
-				Name:        "etcd-snapshot-name",
-				Usage:       "(db) Set the name of etcd snapshots. Default: etcd-snapshot-<unix-timestamp>",
+				Name:        "name",
+				Usage:       "(db) Set the name of the etcd on-demand snapshot. Default: on-demand-<unix-timestamp>",
 				Destination: &ServerConfig.EtcdSnapshotName,
-			},
-			&cli.IntFlag{
-				Name:        "etcd-snapshot-retention",
-				Usage:       "(db) Number of snapshots to retain",
-				Destination: &ServerConfig.EtcdSnapshotRetention,
-				Value:       defaultSnapshotRentention,
+				Value:       "on-demand-" + strconv.Itoa(int(time.Now().Unix())),
```

Comment:
The current timestamp still gets appended to this later, correct?

```suggestion
				Value:       "on-demand-",
```

Reply:
No, which I think relates to the comments above. Timestamp is only used as a suffix when no name is provided. I'll need to  rework some things to have it appended regardless.